plugins/elanfp/fu-elanfp-file-control.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2021
```

Comment:
person or company here please

Reply:
Fixed in commit 2bed27cc885298ea58a2a46baf43a58ab44aebb9